blocks/embed/view.php

```diff
@@ -17,3 +20,9 @@
         </div>
     </div>
 </div>
+<?php
+    }else{?>
+    <a href="<?= h($url) ?>" style="width: 200px; display: block"><?= $url;?></a>
```

Comment:
It's a really good idea to show URL, but I think you don't need to add style for it.
Also, please wrap the url with `h()` function.